### DISCUSSION TITLE
Welcome to polyglot-maven Discussions!

### DISCUSSION THREAD

**Discussion by fbricon** [takari/polyglot-maven#246] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lefou** [comment]
Hi, I'm Tobias, and I maintain the Scala dialect.

**Comment by mojo2012** [comment]
Hi, is this project still alive?

**Reply by cstamas** [reply]
Some parts are maintained, yes. Some are neglected. Furthermore, Maven4 got the needed bits built in, so things will be more interesting once Maven4 out

**Reply by mojo2012** [reply]
I just tried to find what the release plan of maven 4 is. So far I could not find a date. It seems maven 4 has been in development for like forever (over a decade??). Do you have any idea about when we will be able to have a proper release + polyglot support?
